mayland-blocks/block-templates/page.html

```diff
@@ -1,11 +1,17 @@
 <!-- wp:template-part {"slug":"header","align":"full", "tagName":"header","className":"site-header"} /-->
 
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
+
 <!-- wp:spacer {"height":32} -->
 <div style="height:32px" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->
 
 <!-- wp:post-title {"level":1} /-->
 
-<!-- wp:post-content /-->
+</div>
+<!-- /wp:group -->
+
+<!-- wp:post-content {"layout":{"inherit":true}} /-->
```

Comment:
If I don't do this, full width alignment doesn't work inside what's on post-content. I don't know if I'm missing something here.

Reply:
You mean adding layout -> inherit -> true? I think this is the correct way to declare alignment for the post content.

Reply:
No, I mean I had to add a group before and after the post-content block to have all 3 with the inherited layout and having the post-content also working with the full width alignments. If I only add one group block that contains all of it (post title, post content, comments etc), the blocks inside post-content are not direct children of the group that has the layout rules and they never go full wide.

Reply:
Ah I see what you mean. Yeah I still think it's correct even though it requires us to add wrapping containers / group blocks all over the place.

You've probably seen it but there is some related discussion on why this is the case [here](https://github.com/WordPress/gutenberg/issues/29983)

Reply:
Oh I missed that, thank you! then I guess it's fine if we keep doing this in this way, it's not so terrible after all